lib/ansible/modules/identity/ipa/ipa_dnsrecord.py

```diff
@@ -99,6 +99,16 @@
     record_type: 'AAAA'
     record_ip: '::1'
 
+# Ensure a PTR record is present
+- ipa_dnsrecord:
+    ipa_host: spider.example.com
+    ipa_pass: Passw0rd!
+    state: present
+    zone_name: 2.168.192.in-addr.arpa
+    record_name: 5
+    record_type: 'PTR'
+    record_ip: 'internal.ipa.example.com'
```

Comment:
@Akasurde Just so I'm clear, I was proposing renaming all instances of `record_ip` to `record_value`; `record_ip` would cease to exist.

The reason for that being is that `record_ip` makes sense when working with A and AAAA records, but when this module starts supporting all of the other record types, I think `record_value` makes more sense since it is more flexible.

As I said initially, if we were to do this, I think now is the best time since 2.4 still hasn't been released.

I wasn't thinking of having `record_ip` be applicable to A/AAAA records while `record_value` would apply to PTR records.

Are you on the same page?